CHANGELOG.md

```diff
@@ -3,6 +3,46 @@
 Subscribe to the [tracking issue #2231](https://github.com/tigerbeetle/tigerbeetle/issues/2231)
 to receive notifications about breaking changes!
 
+## TigerBeetle 0.16.37
+
+Released: 2025-04-21
+
+### Safety And Performance
+
+- [#2896](https://github.com/tigerbeetle/tigerbeetle/pull/2896)
+
+  Fix a bug where VOPR latencies were computed incorrectly when the minimum
+  and mean were equal.
+
+### Internals
+
+- [#2893](https://github.com/tigerbeetle/tigerbeetle/pull/2893)
+
+  Make the `stdx.StackType(T)` intrusive pointer non-generic (`*StackLink`) over the
+  pointee type (`*T`), while keeping the overall API generic for type safety.
```

Comment:
Let me take a second attempt at writing a useful commit message here: 

> Improve binary size and compilation time by making implementation of intrusive stack non-generic.

For commit messages, I tend to more "what" the summary line, as, contrary to the popular opinion, "why" tends to be less useful than "what" there. You usually look at commit messages to find a change that broke something, and there "what" is more useful as it has higher chance to capture unintended consequences. And, in context, "why" usually follows. 

For user-facing changelings, we should lean more on the why side.